indra/statements/statements.py

```diff
@@ -195,6 +207,9 @@
     'Complex', 'Translocation', 'RegulateAmount', 'DecreaseAmount',
     'IncreaseAmount', 'Influence', 'Conversion', 'Unresolved',
     'Association', 'Event', 'Migration',
+    'Carboxylation', 'Decarboxylation', 'Amidation', 'Deamidation', 'Sulfation',
```

Comment:
I'm trying to think of any other places we need to account for these statement types. One would be the INDRA Statements JSON specification but there may be more.

Reply:
The new statement types are now in a separate branch, PRd here: #1161